Jenkinsfile

```diff
@@ -23,6 +23,9 @@ pipeline {
 			steps {
 				sh 'cargo test --verbose --all'
 				sh 'cargo test --verbose --all --release'
+                sh 'cargo test --manifest-path eth2/state_processing/Cargo.toml --verbose \
+                               --release --features fake_crypto --ignored'
```

Comment:
This should actually be `--include-ignored` (because `--ignored` will only run the ignored tests)

Reply:
And it should also be `-- --include-ignored` as its causing Jenkins to fail.